typehints/common/xbee/__init__.pyi

```diff
@@ -30,7 +30,7 @@ PROFILE_DIGI_XBEE: int = ...
 PIN_WAKE: int = ...
 RTC_WAKE: int = ...
 
-def atcmd(cmd: str, value: Optional[Any]) -> Optional[Any]:
+def atcmd(cmd: str, value: Optional[Any]=None) -> Optional[Any]:
```

Comment:
Generally the code style when specifying default arguments along with a type annotation is to surround the `=` with spaces. (I know, that's kind of against PEP8 style, but it is convention.)

```python
def atcmd(cmd: str, value: Optiona[Any] = None) -> Optional[Any]:
    ...
```

Reply:
I believe [PEP484 ](https://www.python.org/dev/peps/pep-0484/) expands upon and supersedes PEP8, as appropriate.